### PR TITLE
Techtonica support, closes #1171

### LINKED ISSUE #1171: Techtonica
https://discord.com/channels/806885678201831424/1149419302827589632

### PR DESCRIPTION
**Pull request by FreddecGames**
Added entries for Techtonica game
Added Techtonica game data and icons

### CODE REVIEW

**Comment by dcbroad3** [comment]
Could you increment the version in `package.json`? The next version should be 2.3.93 (main was just updated a minute ago, so it might show as 2.3.91 currently in this branch)

**Comment by dcbroad3** [comment]
Also, looks like some tests need to be cleaned up in this branch. The current test error is indicating that mocks need to be updated, although there may be other problems.

File: src/app/models/game-info.ts [hunk 1]
```diff
@@ -45,4 +45,12 @@ export const gameInfo: Record<Game, GameInfo> = {
     modId: 'coi',
     hideColumns: ['beacons', 'pollution', 'power', 'wagons'],
   },
+  [Game.Techtonica]: {
+    icon: 'techtonica',
+    route: 'techtonica',
+    label: 'options.game.techtonica',
+    meta: 'Techtonica',
+    modId: 'tta',
+    hideColumns: [],
```
Comment (dcbroad3) [inline src/app/models/game-info.ts:54]: Does this data include information for all columns? I would think we would at _least_ hide the `beacons` column. I haven't inspected the data closely, but you may also want to hide `pollution`, `power`, and/or `wagons`.

File: src/data/tta/icons.png
Comment (dcbroad3) [inline src/data/tta/icons.png:1]: Don't need to include the `png` version of the icons, only the `webp`

File: src/app/models/game-info.ts (hunk 1 above)
Comment (FreddecGames) [reply src/app/models/game-info.ts]: Fixed

File: src/data/tta/icons.png
Comment (FreddecGames) [reply src/data/tta/icons.png:1]: Fixed

**Pull request merged**